index.html

```diff
@@ -135,6 +135,32 @@ <h4>EditContext state</h4>
               from the [=Text Input Service=]. The user agent will indicate which positions are
               required by firing {{CharacterBoundsUpdateEvent}}.
             </p>
+            <p>
+              [=Control bounds=], [=selection bounds=], and [=codepoint rects=] are given in the
+              <dfn>client coordinate system</dfn>, which is defined as a two-dimensional Cartesian
+              coordinate system (x, y) where the origin is the top-left corner of the viewport,
```

Comment:
> viewport

You mean the layout viewport? If so, can we link it to https://drafts.csswg.org/cssom-view/#layout-viewport?

Reply:
or is it a more general definition of viewport: https://www.w3.org/TR/CSS21/visuren.html#x1?

Reply:
I think layout viewport is correct here. Made these into links.